src/databricks/sql/__init__.py

```diff
@@ -5,7 +5,10 @@
 # PEP 249 module globals
 apilevel = "2.0"
 threadsafety = 1  # Threads may share the module, but not connections.
-paramstyle = "named"  # Python extended format codes, e.g. ...WHERE name=%(name)s
+
+# Python extended format codes, e.g. ...WHERE name=%(name)s
+# Note that when we switch to ParameterApproach.NATIVE, paramstyle will be `named`
+paramstyle = "pyformat"  
```

Comment:
What is the use of `paramstyle` here?
Should we say: inline = pyformat while native = named?
I think I am not very clear on this.